install_config/aggregate_logging.adoc

```diff
@@ -420,6 +420,13 @@ for deploying Kibana instances.
 |`openshift_logging_curator_ops_nodeselector`
 |A node selector that specifies which nodes are eligible targets
 for deploying Curator instances.
+
+|`openshift_logging_purge_logging`
+|When `openshift_logging_install_logging` is set to 'false' to trigger
+uninstalation and `openshift_logging_purge_logging` is set to 'true', it will
```

Comment:
Not sure what "it" is here (but, maybe our end-users will). Same with "trigger uninstallation here and with the `openshift_logging_install_logging` description above. 
I like the way the flag is described in the dev PR as it has a bit of a user story. Maybe we don't need it. 
https://github.com/openshift/openshift-ansible/pull/5189

Reply:
I meant "it" to refer to the ansible playbook used to install or uninstall logging. Feel free to rewrite the description of the `openshift_logging_purge_logging` in a clearer and more understandable way but I think the dev PR text is unnecessarily long for the docs description. The only reason I was so verbose there is that PR text doesn't have any context while the description in docs is surrounded by rich context.